src/core.jl

```diff
@@ -1,35 +1,99 @@
+"""
+    JSONRPCError
+
+An object representing a JSON-RPC Error.
+
+Fields:
+ * code::Int
+ * msg::AbstractString
+ * data::Any
+
+See Section 5.1 of the JSON RPC 2.0 specification for more information.
+"""
 struct JSONRPCError <: Exception
     code::Int
     msg::AbstractString
     data::Any
 end
 
+"""
+    SERVER_ERROR_END
+
+The end of the range of server-reserved errors.
+
+These are JSON-RPC server errors that are free for the taking
+for JSON-RPC server implementations. Applications making use of
+this library should NOT define new errors in this range.
+"""
+const SERVER_ERROR_END       = -32000
```

Comment:
Nitpick, but can you fix the whitespace here and below pls?

Reply:
Sure, that was just leftover from when they were defined together in a bigger list.